### ISSUE TITLE
First write on new database hangs (0.9.0_rc30)

### ISSUE THREAD

**Issue by kpweiler** [influxdb/influxdb#2619]
I've written a python script that pulls data from impala, and writes it to influxdb (~ 1000000 rows) in batches of 10k. I'm using the influxdb python module to write. For some reason, when I bring influxd up, create a database, and run my script, it hangs after sending the first 10k batch. If, instead, I send a single row as my first write, and then try to run my 10k batch script, everything writes fine. I did an strace on my script and see that it seems the client is waiting for a response from the server that it's not getting:

```
socket(PF_INET, SOCK_STREAM, IPPROTO_TCP) = 5
setsockopt(5, SOL_TCP, TCP_NODELAY, [1], 4) = 0
fcntl(5, F_GETFL)                       = 0x2 (flags O_RDWR)
fcntl(5, F_SETFL, O_RDWR)               = 0
connect(5, {sa_family=AF_INET, sin_port=htons(8086), sin_addr=inet_addr("127.0.0.1")}, 16) = 0
sendto(5, "POST /write?p=integra%29&u=trad_"..., 2804107, 0, NULL, 0) = 2804107
fcntl(5, F_GETFL)                       = 0x2 (flags O_RDWR)
fcntl(5, F_SETFL, O_RDWR)               = 0
recvfrom(5,
```

Hope this isn't a duplicate of an existing issue, but I couldn't seem to find anything that matched.

**Comment by jwilder** [comment]
This was recently fixed by #2610.  It's in master and should be in the next release.

**Comment by kpweiler** [comment]
Thanks! Closing now.
